src/cheatman.c

```diff
@@ -351,7 +355,7 @@ int load_cheats(const char *cheatfile)
     ret = parse_buf(buf);
     free(buf);
     if (ret < 0)
-        return -1;
+        return (text_ret == -ENOENT) ? text_ret : -1;
```

Comment:
Doh, I am not so advanced but tgis line looks suspicious. Maybe other @ps2homebrew can look at this?

Reply:
> Doh, I am not so advanced but this line looks suspicious. Maybe other @ps2homebrew can look at this?

Looks suspicious too

Did that on cellphone

Will correct that when I have time